cli/codegen/generate_code.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/dave/jennifer/jen"
+)
+
+type generateLuaCodeOpts struct {
+	PackageName  string
+	FileName     string
+	PackagePath  string
+	VariablesMap map[string]string
+}
+
+var luaCodeFiles = []generateLuaCodeOpts{
+	generateLuaCodeOpts{
+		PackageName: "admin",
+		FileName:    "cli/admin/lua_code_gen.go",
+		VariablesMap: map[string]string{
+			"adminListFuncBodyTmpl":  "cli/admin/lua/admin_list_func_body_template.lua",
+			"evalFuncGetResBodyTmpl": "cli/admin/lua/eval_func_get_res_body_template.lua",
+			"funcHelpMsgTmpl":        "cli/admin/lua/func_help_msg_template.lua",
```

Comment:
It's not Lua =)

BTW, I think that it's one of templates that we can continue to store in string variable. It's instantiated in runtime and it isn't Lua code. One of reasons we separate Lua code strings to files - it's hard to edit Lua code in string without syntax highlight.